pkg/datapath/linux/node.go

```diff
@@ -562,11 +562,11 @@ func (n *linuxNodeHandler) getSrcAndNextHopIPv4(nodeIPv4 net.IP, ifaceName strin
 	// Figure out whether nodeIPv4 is directly reachable (i.e. in the same L2)
 	routes, err := netlink.RouteGet(nodeIPv4)
 	if err != nil {
-		return nil, nil, fmt.Errorf("failed to retrieve route for remote node IP: %w", err)
+		return nil, nil, fmt.Errorf("Failed to retrieve route for remote node IP: %w", err)
 	}
 
 	if len(routes) == 0 {
-		return nil, nil, fmt.Errorf("remote node IP is non-routable")
+		return nil, nil, fmt.Errorf("Remote node IP is not routable. Connectivity to pods on that node may be unavailable.")
```

Comment:
I think this commit is reverting all changes from the two previous commits by mistake.

Reply:
:facepalm:

Reply:
I don't think this is resolved. Most of commit `node-neigh: Unconditionally remove neigh entry from BPF map` is just reverts of the two previous commits.

Reply:
:facepalm: x2 (having two workdirs doesn't help)